### ISSUE TITLE
Change animation type

### ISSUE THREAD

**Issue by blackstormx** [ganlanyuan/tiny-slider#520]
I want tiny slider to slide into one direction continuously without stopping but with infinite looping enabled, is there any way I can achieve this? I know its possible with owl- carousel, but can't seem to find out how to do it with tiny-slider.

Editing source files is also an option.

Thanks for any help.

**Comment by manouman** [comment]
Did you find a solution for what you mentioned here @blackstormx  ?

**Comment by blackstormx** [comment]
> Did you find a solution for what you mentioned here @blackstormx  ?

Yes, i ended up implementing my own slider with JS, it wasn't too hard. Tpis: Use css tranaforms and requestanimationframe for that.

**Comment by manouman** [comment]
hehe , all right man.
thanks!

**Comment by johncarter-** [comment]
I'm looking for the way to do this too. Wanted to create a continuous infinite loop, like a ticker. Any idea?

**Comment by lachlanstephen** [comment]
Bit late to the party but set `transition-timing-function` to `linear !important` in the CSS for the slider. Then use the following options in the tns() call:

```autoplay: true,
loop: true,
speed: <whatever value you want>,
autoplayTimeout: 0,
mouseDrag: false,
touch: false,
preventActionWhenRunning: true
```

Something to note is slides need to be the same width otherwise the speed will ramp up and down to conform to the speed option. 

Also the `autoplayHoverPause` breaks slightly as it will only pause when the next image gets to the starting position. Generally it'll work as a scrolling gallery but any mouse or touch interaction breaks a bit.

**Comment by lachlanstephen** [comment]
After a bunch of testing, I think the easiest way to do it is a CSS only way. This video [here](https://www.youtube.com/watch?v=KD1Yo8a_Qis) explains it well.
